storage/ndb/rest-server/rest-api-server/internal/integrationtests/batchpkread/performance_test.go

```diff
@@ -0,0 +1,147 @@
+/*
+ * This file is part of the RonDB REST API Server
+ * Copyright (c) 2023 Hopsworks AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package batchpkread
+
+import (
+	"math/rand"
+	"net/http"
+	"sort"
+	"testing"
+	"time"
+
+	"google.golang.org/grpc"
+	"hopsworks.ai/rdrs/internal/config"
+	"hopsworks.ai/rdrs/internal/integrationtests/testclient"
+	"hopsworks.ai/rdrs/internal/testutils"
+	"hopsworks.ai/rdrs/pkg/api"
+	"hopsworks.ai/rdrs/resources/testdbs"
+)
+
+/*
+	The number of parallel client go-routines spawned can be influenced by setting
```

Comment:
Bit confusing, GOMAXPROCS only controls the number of runtime threads. These threads can handle any number of go routines.